src/main/java/com/criticalsoftware/Contact.java

```diff
@@ -0,0 +1,49 @@
+package com.criticalsoftware;
+
+import io.quarkus.mongodb.panache.PanacheMongoEntity;
+
+public class Contact extends PanacheMongoEntity {
+
+    private String email;
+    private int phoneNumber;
+    private String address;
+
+    public Contact() {
+
```

Comment:
Can we try without the empty constructor?

Reply:
Doesn't work without